.pre-commit-config.yaml

```diff
@@ -31,3 +31,8 @@ repos:
   hooks:
   - id: pyupgrade
     args: ["--py36-plus"]
+- repo: https://github.com/crate-ci/typos
```

Comment:
Suggest separating this commit into a new one.

Reply:
Hello @luckyh, do you mean an individual patch? If yes, I will drop it this time and into a new.

Reply:
@PaulYuuu yeah, I mean, a new PR. Sorry for the confusion.

Reply:
OK, will do that!